winui/DropDown-Color-Palette/Getting-Started.md

```diff
@@ -146,7 +146,7 @@ SfDropDownColorPalette sfDropDownColorPalette =  new SfDropDownColorPalette();
 {% endhighlight %}
 {% endtabs %}
 
-![Dropdown Color Palette control with theme and standard color items](Getting-Started_images/ThemeColorPanel.png)
+![WinUI Dropdown Color Palette control with theme and standard color items](getting-started_images/winui-color-palette-theme-colo-panel.png)
```

Comment:
image name is wrong. check properly.
colo-panel

Reply:
Renamed